pkg/config/autodiscover/csv_info.go

```diff
@@ -0,0 +1,68 @@
+package autodiscover
```

Comment:
Please add a copyright header.

Reply:
Seems like I have a blind spot! Copying the wrong one last time, omitting it completely this time. Thanks for spotting it!